src/pure/cmd_analysis.ml

```diff
@@ -92,10 +134,17 @@ let qidents_of_cmd (cmd : t) =
   | Syntax.P_require_as (_, _) -> []
   | Syntax.P_open _ -> []
   | Syntax.P_rules rules -> concat_map qidents_of_p_rule rules
-  | Syntax.P_symbol {p_sym_arg;p_sym_typ;p_sym_trm;_} ->
+  | Syntax.P_symbol {p_sym_arg;p_sym_typ;p_sym_trm;p_sym_prf;_} ->
     let some_or_empty = function Some arg -> [arg] | None -> [] in
+    let prfqidlist = begin
+      match p_sym_prf with
+      | Some (ptac_list, _) -> List.concat_map qidents_of_p_tactic ptac_list
+      | None -> []
+    end in
     let terms_list = some_or_empty p_sym_typ @ some_or_empty p_sym_trm in
-    filter_bound_qidents p_sym_arg terms_list
+    let res =  (filter_bound_qidents p_sym_arg terms_list) in
```

Comment:
spaces and parentheses useless here, and lines 145-146 can perhaps be put on a single line:
`filter_bound_qidents p_sym_arg terms_list @ prfqidlist`

Reply:
Done wherever changes were possible